eo-maven-plugin/src/test/resources/org/eolang/maven/binarize/eo_env/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "eo_env"
+version = "0.1.0"
+edition = "2021"
```

Comment:
@levBagryansky Is it a typo?

Reply:
@Graur what's wrong?

Reply:
@levBagryansky Edition is 2021. Is it ok?

Reply:
@Graur According to https://doc.rust-lang.org/cargo/getting-started/first-steps.html its ok since command `cargo new <new_project>` set this value.

Reply:
```
tardis3@RyanGosling:~/rust_trash$ cargo new to_remove
     Created binary (application) `to_remove` package
tardis3@RyanGosling:~/rust_trash$ cd to_remove/
tardis3@RyanGosling:~/rust_trash/to_remove$ ls
Cargo.toml  src
tardis3@RyanGosling:~/rust_trash/to_remove$ cat Cargo.toml 
[package]
name = "to_remove"
version = "0.1.0"
edition = "2021"

# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html

[dependencies]
tardis3@RyanGosling:~/rust_trash/to_remove$
```